addons/web_editor/static/src/less/web_editor.ui.less

```diff
@@ -653,6 +677,13 @@ html[lang] > body.editor_enable [data-oe-translation-state] {
                     }
                 }
 
+                &.o_disable_btn {
```

Comment:
Maybe we should just toggle the `readonly` class instead of a new one ?

Reply:
as FGI said; the behavior of +/- button when limit (min 1 and max 6) is exceed we have to make it gray and disable; dont hide button.
while readonly will hide the button that's why i created new class.